ca8467c41b9abc10ce0f62c3b24bcbaa/author_elective_update/scan_column_type_check_176c32c097123110839d76021153aff4.xml

```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0" encoding="UTF-8"?><record_update table="scan_column_type_check">
+    <scan_column_type_check action="DELETE">
```

Comment:
@Irston Why is this trying to delete a scan?

Reply:
I had created duplicates and deleted one, technically it shouldn't really delete anything in the main repo @niamccash

Reply:
The PR only contains the deleted one. I'm not seeing the (new) one you added. Am I missing something?

Reply:
@niamccash Apologies, new to using GIT so i messed up branches, I raised a separate PR and checked to see if all okay this time around.
Sorry for the trouble.

Reply:
@Irston Don't sweat it. I hope things are starting to make sense and you're learning something as you participate. We appreciate your contributions and participation. Will review your new PR as soon as I can.